src/zen/glance/ZenGlanceManager.mjs

```diff
@@ -116,6 +119,52 @@
         .getElementById('zen-glance-sidebar-template')
         .content.cloneNode(true);
       const container = newButtons.querySelector('.zen-glance-sidebar-container');
+      // Set helpful tooltips, including the current shortcut if available
+      try {
+        const closeBtn = newButtons.querySelector('.zen-glance-sidebar-close');
+        const expandBtn = newButtons.querySelector('.zen-glance-sidebar-open');
+        const splitBtn = newButtons.querySelector('.zen-glance-sidebar-split');
+
+        const buildShortcutText = (keyElem) => {
```

Comment:
This should rather be moved into a shortcut generic space, like on nsZenKeyboardShortcut...